velox/docs/functions/iceberg/functions.rst

```diff
@@ -23,7 +23,34 @@ Refer to `Iceberg documenation <https://iceberg.apache.org/spec/#partition-trans
        SELECT bucket(128, 'abcd'); -- 4
        SELECT bucket(100, 34L); -- 79
 
+.. iceberg:function:: days(input) -> integer
+
+   Returns the day of the month from a date or timestamp input, as days from 1970-01-01.
+
+   Supported types for ``input`` are: ``DATE``, ``TIMESTAMP``. ::
```

Comment:
This seems to be a duplicate of the above, since the previous document already clarifies the input types. Do you think it makes sense to remove this?